.github/workflows/e2e-tests.yml

```diff
@@ -10,6 +10,9 @@ on:
 
 jobs:
   kubernetes-test:
+    strategy: 
+      matrix:
+        k8s-version: ["v1.26.0", "v1.25.3", "v1.24.7", "v1.23.13"]
```

Comment:
Do we need to specify the patch version or can be `v1.26` etc?

Reply:
https://github.com/kubernetes-sigs/kind/releases

I got the supported versions in this page, and did not see other version in 1.26 .

I also check the kind official docker hub, only `1.26.0` is available.

Reply:
Ok, sounds good. Thanks for confirming